site/en/blog/extension-safety-hub/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Bringing Safety Check to the chrome://extensions page"
+description: "Starting in Chrome 117, Chrome will proactively flag to users when an extension they have installed is no longer in the Chrome Web Store."  
```

Comment:
I'm not sure the new wording is correct - I was using "flag" to mean "indicate to the user" rather than showing a literal flag. Would the following work?
```suggestion
description: "Starting in Chrome 117, Chrome will proactively highlight to users when an extension they have installed is no longer in the Chrome Web Store."  
```

Reply:
SGTM